lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -8,6 +8,45 @@
     position: relative;
   }
 
+  &.-fullscreen {
+    position: fixed;
+    z-index: 1000;
+    top: 0;
+    left: 0;
+    width: 100vw;
```

Comment:
Yep yep. I checked the caniuse.com info, and seems it's only partial support for `vmax` so I think we'd be in the clear with `vw` but if we're still targeting IE8 then defs not...